ufl/action.py

```diff
@@ -159,29 +157,36 @@ def __hash__(self):
 
 def _check_function_spaces(left, right):
```

Comment:
Dear code reviewers,

Please ignore the rest of the diff, that's just ruff.
Only take a look at this function.

Thank you.